master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
```

Comment:
This doc doesn't have a "When should I read this doc" type intro - I think we should be aiming to have that sort of introduction for all of our usage guides.